data/i18n/en/en.toml

```diff
@@ -337,6 +337,9 @@ other = "Options"
 [outdated_blog__message]
 other = "This article is more than one year old. Older articles may contain outdated content. Check that the information in the page has not become incorrect since its publication."
 
+[outdated_release_info_warning]
+other = " documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For the latest release information, please refer to the "
```

Comment:
Partial sentences are very hard to localize. Please consider splitting this up into something where the localized strings are whole sentences.